include/tvm/runtime/container.h

```diff
@@ -188,6 +188,709 @@ class InplaceArrayBase {
     return data_start + idx * sizeof(ElemType);
   }
 };
+}  // namespace runtime
+}  // namespace tvm
```

Comment:
We don't need to introduce a new namespace bracket, as both are part of the same tvm::runtime